packages/recurring-payments-job/src/recurring-payments-processor.js

```diff
@@ -1,3 +1,12 @@
-export async function execute () {
-  // do something
+import { getRecurringPayments } from '../../sales-api-service/src/services/recurring-payments.service.js'
```

Comment:
You shouldn't be calling Sales API modules in this way. They should be called via the HTTP interface - you can't rely on the sales-api-service being available, as when recurring-payments is running in its own container it won't be. For an example of how to do it, have a look at the agreed-handler.js in the GAFL. You'll see there's a salesApi import there, that actually connects to the @defra-fish/connectors-lib, which in turn makes the HTTP request to the Sales API